distribution/linux/openrct2.appdata.xml

```diff
@@ -50,65 +64,99 @@
       określonego celu w wyznaczonym czasie, podczas gdy tryb sandbox pozwala na
       swobodne budowanie parku, z możliwością wyłączenia ograniczeń i finansów.
     </p>
+    <p xml:lang="pt">
+      OpenRCT2 é uma reimplementação de código aberto do RollerCoaster Tycoon 2 (RCT2).
+      A dinâmica de jogo gira em torno de construir e manter um parque de diversões contendo
+      atrações, lojas e instalações. O jogador deve tentar gerar lucro e manter uma boa
+      reputação para o parque, enquanto tenta fazer com que os visitantes continuem felizes.
+      O OpenRCT2 permite tanto jogos em cenários quanto em "caixa de areia". Os cenários
```

Comment:
Could you use non-typewriter quotes here?

Reply:
Done, I think. This always confuses me.